vercel.json

```diff
@@ -914,7 +914,6 @@
             "source": "/questions/what-does-pageview-means-scattered-across-the-site",
             "destination": "/docs/data/events"
         },
-        { "source": "/docs/data/data-warehouse", "destination": "/docs/data-warehouse" },
```

Comment:
I think it makes sense to keep this, right? I assume this was a redirect for the docs themselves.

Reply:
Yep, my bad